### PR TITLE
test: add tests to `blas/ext/base/dnanasum`

### LINKED ISSUE #3046: Incomplete code coverage for `blas/ext/base/dnansumkbn`
See https://coverage.stdlib.io/blas/ext/base/dnansumkbn/index.html.

This should be straightforward to address. Just need to add a test case in each of the test files which handles the situation where the stride is `0` and the element is `NaN`.

### PR DESCRIPTION
**Pull request by saurabhraghuvanshii**
## Description

> What is the purpose of this pull request?

This pull request:

-   Add test case for blas/ext/base/dnansumk and dnansumk-native.js

## Related Issues

> Does this pull request have any related issues?

This pull request is related to:

-  #3046 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-  [x] Read, understood, and followed the [contributing guidelines](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

* * *

@stdlib-js/reviewers

### CODE REVIEW

**Comment by saurabhraghuvanshii** [comment]
check the contributing guidelines

**Comment by saurabhraghuvanshii** [comment]
accidently closed PR

**Comment by kgryte** [comment]
@saurabhraghuvanshii Thank you for opening this PR; however, as is, we cannot accept it. The current PR makes several undesired stylistic changes and does not follow our style guidelines or the dominant conventions used throughout the project.

While the PR does add additional tests, those cannot be merged until the various stylistic changes are reverted.

**Comment by saurabhraghuvanshii** [comment]
Hii @kgryte if I follow your guidelines of styles and improved it , can be it merged , and sir by mistake I use prettier which change some style.

**Comment by kgryte** [comment]
@saurabhraghuvanshii Yes, this PR should be on the pathway to being merged. However, you should also see https://github.com/stdlib-js/stdlib/pull/3051/files which implements a very similar set of changes. You should replicate those changes here, but for `blas/ext/base/dnansum`.

**Comment by saurabhraghuvanshii** [comment]
@kgryte ok sir

**Comment by saurabhraghuvanshii** [comment]
@kgryte sir , change blas/ext/base/dnansum by refrence #3051 , If any change requires I will do 
and Thanks for your valuable time .

**Comment by saurabhraghuvanshii** [comment]
@kgryte back to Previous Style

File: lib/node_modules/@stdlib/blas/ext/base/dnanasum/test/test.dnanasum.js [hunk 1]
```diff
@@ -187,4 +199,4 @@ tape( 'the function supports view offsets', function test( t ) {
 	t.strictEqual( v, 9.0, 'returns expected value' );
 
 	t.end();
-});
+});
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dnanasum/test/test.dnanasum.js]:
```suggestion
});

```

File: lib/node_modules/@stdlib/blas/ext/base/dnanasum/test/test.dnanasum.native.js [hunk 2]
```diff
@@ -278,4 +290,4 @@ tape( 'the function supports view offsets', opts, function test( t ) {
 	t.strictEqual( v, 9.0, 'returns expected value' );
 
 	t.end();
-});
+});
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/dnanasum/test/test.dnanasum.native.js]:
```suggestion
});

```

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @saurabhraghuvanshii.

**Comment by saurabhraghuvanshii** [comment]
@kgryte thanks sir

**Pull request merged**